### DISCUSSION TITLE
HTTP 1

### DISCUSSION THREAD

**Discussion by gachikuku** [mitmproxy/mitmproxy#7015] [General]
Hello, 

I visit a [HTTP/1.1 Test Web Site](https://www.w3.org/Protocols/HTTP/Performance/microscape/), and there is no content displayed in the *Response*.
There should be content. How do I modify `mitmproxy` to display content for non HTTP/2 websites.

I was really satisfied with the answers to my previous question.

Thank you and sorry for bothering.

**Comment by gachikuku** [comment]
When I do 
```sh
curl -x http://localhost:8080 https://www.w3.org/Protocols/HTTP/Performance/microscape/
```
There is content displayed in the *Response* on `mitmproxy`
But when I visit the same website via Chrome (using Shift + B to open the browser using `mitmproxy`) It does not work.

```
Flows
>>04:11:01 HTTPS GET          www.w3.org /Protocols/HTTP/Performance/microscape/                                   200       text/html  3.7k 204ms
  04:11:04 HTTPS GET          www.w3.org /Protocols/HTTP/Performance/microscape/                                   304    [no content]       178ms
```

**Comment by Prinzhorn** [comment]
`304` -> Chrome has the page in cache already. Use `mitmproxy --anticache` or whatever method you prefer (e.g. open dev tools and disable caching or make a force refresh or use a new private browsing window)
